src/dstr-assignment/obj-rest-computed-property.case

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2017 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: >
+    Destructuring field can be a computed property, i.e it can be defined 
+    only at runtime. Rest operantion needs to skip these properties as well.
+template: default
+esid: pending
+includes: [propertyHelper.js]
+features: [object-rest]
+---*/
+
+//- setup
+var a = "foo";
+//- elems
+{[a]:b, ...rest}
+//- vals
+{ foo: 1, bar: 2, baz: 3 }
+//- body
+assert.sameValue(b, 1);
+assert.sameValue(rest.bar, 2);
+assert.sameValue(rest.baz, 3);
+
+verifyEnumerable(rest, "bar");
+verifyWritable(rest, "bar");
+verifyConfigurable(rest, "bar");
```

Comment:
Cool. That's fair. The problem with binding is the scope cases in the templates. One example of case that will be generated but it isn't valid is:

```
let a = "foo"

function b({[a]: c, ...r}) {
//...
}

b({foo: 2, bar: 3});
```
Do you have any workaround in mind? I could add "let" cases manually, BTW.